src/main/java/com/facebook/openwifirrm/modules/operationelement/HTOperationElement.java

```diff
@@ -0,0 +1,162 @@
+package com.facebook.openwifirrm.modules.operationelement;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+import org.apache.commons.codec.binary.Base64;
+
+import com.facebook.openwifirrm.Utils;
+
+/**
+ * High Throughput (HT) Operation Element, which are potentially present in
+ * wifiscan entries. Details in IEEE 802.11-2020 standard.
+ */
+public class HTOperationElement implements OperationElement {
+
+	protected final byte primaryChannel;
+	protected final byte secondaryChannelOffset;
+	protected final boolean staChannelWidth;
+	protected final boolean rifsMode;
+	protected final byte htProtection;
+	protected final boolean nongreenfieldHtStasPresent;
+	protected final boolean obssNonHtStasPresent;
+	protected final byte channelCenterFrequencySegment2;
+	protected final boolean dualBeacon;
+	protected final boolean dualCtsProtection;
+	protected final boolean stbcBeacon;
+	protected final byte[] basicHtMcsSet;
+
+	/**
+	 * XXX some combinations of these parameters may be invalid as defined by
+	 * 802.11-2020, but this is not checked here. If fidelity to 802.11 is required,
+	 * the caller of this method must make sure to pass in valid parameters. The
+	 * 802.11-2020 specification has more details about the parameters.
+	 *
+	 * @param primaryChannel                 channel index
```

Comment:
fix spacing

Reply:
This is the correct spacing (using the style that all param descriptions should start to the right of ALL params, so in this case it starts to the far right because `channelCenterFrequencySegment2` is so long)